main.go

```diff
@@ -143,12 +144,50 @@ func runElectron() {
 		return
 	})
 	w.On(astilectron.EventNameWindowEventDidFinishLoad, func(e astilectron.Event) (deleteListener bool) {
-		log.Infoln("[axolotl-electron] Electron App load")
-		w.ExecuteJavaScript("window.onToken = function(token){window.location='http://" + config.ServerHost + ":" + config.ServerPort + "/?token='+token;};")
+		log.Infoln("[axolotl-electron] Page loaded")
 		return
 	})
 	w.On(astilectron.EventNameWindowEventWillNavigate, func(e astilectron.Event) (deleteListener bool) {
-		log.Infoln("[axolotl-electron] Electron navigation")
+		log.Infoln("[axolotl-electron] Electron navigation", e.URL)
+		if strings.Contains(e.URL, "signalcaptchas.org") {
+			log.Infoln("[axolotl-electron] overriding onload", e.URL)
+
+			w.ExecuteJavaScript(
+				`
```

Comment:
If I understand this right, in all three places we are executing javascript. Could we extract this JS snippet into a separate file, and import it into the three files? Then if we need to do any later adjustments we only need to do them once.